hudi-client/src/main/java/org/apache/hudi/config/HoodieCompactionConfig.java

```diff
@@ -147,11 +147,6 @@ public Builder withInlineCompaction(Boolean inlineCompaction) {
       return this;
     }
 
-    public Builder inlineCompactionEvery(int deltaCommits) {
-      props.setProperty(INLINE_COMPACT_PROP, String.valueOf(deltaCommits));
```

Comment:
this is wrong nonetheless.. we can remove this method.. 

 but the issue you faced may not be due to this? I see `LogFileSizeBasedCompactionStrategy` is the default strategy, which chooses the largest log files for compaction (like I mentioned, we check if compaction has to be scheduled at the end of each delta commit), upto a certain size .. This default is very large (probably due to the data volumes at uber)// 

`  public static final String DEFAULT_TARGET_IO_PER_COMPACTION_IN_MB = String.valueOf(500 * 1024);` 

And it keeps compacting all the logs, thus.. 

Could you try setting `hoodie.compaction.target.io=1` (i.e allow upto just 1MB, artifically low threshold) and see if the log files are there.. We can also revisit if we can lower the limit for that in this PR.

Reply:
@sathyaprakashg : Thanks for catching this.
Yes, this method should not be there in the first place. 
Though, For context, this is innocuous as Hudi uses the property hoodie.compact.inline.max.delta.commits for figure out inline compaction interval. The associated set API  is withMaxNumDeltaCommitsBeforeCompaction()(https://hudi.apache.org/docs/configurations.html#withMaxNumDeltaCommitsBeforeCompaction). Can you try setting this value : hoodie.compact.inline.max.delta.commits to a higher value ?

Reply:
Thanks @vinothchandar and @bvaradar for the information.  
I see that currently, default value of hoodie.compact.inline.max.delta.commits is 1 and it is making merge on read table to do inline compaction during every load. We can set custom value in specific load, but it could be better if default value is higher than 1, so that MOR table does not behave like COW table by recreating affected parquet file every time.

I created MR for this. Please review and let me know your thoughts.
https://github.com/apache/hudi/pull/1664